lightning/src/ln/channel.rs

```diff
@@ -2148,6 +2149,13 @@ impl<ChanSigner: ChannelKeys> Channel<ChanSigner> {
 									match e {
 										ChannelError::Ignore(ref msg) => {
 											log_info!(logger, "Failed to send HTLC with payment_hash {} due to {}", log_bytes!(payment_hash.0), msg);
+											// If we fail to send here, then this HTLC should
```

Comment:
It'd be nice to have a test which has more than just this one thing in the holding cell - ie, that we can for example claim an HTLC in the same holding-cell-free call as we fail to forward one.